lib/src/components/NxSmallThreatCounter/__tests__/NxSmallThreatCounter.test.tsx

```diff
@@ -4,280 +4,157 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
-import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
-import 'jest-enzyme';
-import NxTooltip from '../../NxTooltip/NxTooltip';
+import { RenderResult } from '@testing-library/react';
+import { waitFor } from '@testing-library/react';
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
 
 import NxSmallThreatCounter, { Props } from '../NxSmallThreatCounter';
 
 describe('NxSmallThreatCounter', function() {
-  const minimalProps = {},
-      getMountedComponent = enzymeUtils.getMountedComponent<Props>(NxSmallThreatCounter, minimalProps);
+  const quickRender = rtlRender<Props>(NxSmallThreatCounter, {}),
+      renderEl = rtlRenderElement<Props>(NxSmallThreatCounter, {});
+
+  function getCounter(component: RenderResult, category: string) {
```

Comment:
Probably best to use `getByTitle` for these

Reply:
Agree, it has the title attribute.
Updated here 90384b82c1b3e088c4144729e663709de552c392, thanks.